app/components/searchable_dropdown_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class SearchableDropdownComponent < ViewComponent::Base
```

Comment:
Great to see the code re-usable. I think I would have created an option on the Stimulus TomSelectController to do this easier in Javascript. 
But we know the number of options already on the server, so we can make the choice before getting to the browser 👍